src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -150,14 +150,20 @@ public function getServerParameter($key, $default = '')
         return isset($this->server[$key]) ? $this->server[$key] : $default;
     }
 
-    public function switchToXHR()
+    /**
+     * Calls a URI with XMLHttpRequest.
+     *
+     * @return Crawler
```

Comment:
This could be removed in favor of using `: Crawler` in the method signature instead.